script/show-sysinit.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import os.path
+import sys
+import subprocess
+
+if __name__ == "__main__":
+  print('digraph mimiker_modules {')
+  for folder in ['sys','mips']:
+    command='grep {}/* -e SYSINIT_ADD'.format(folder)
+    out=subprocess.check_output(command,shell=True)
+    out=out.decode('UTF-8')
+    for line in out.split('\n'):
+      if not 'SYSINIT_ADD' in line:
+        continue
+      line=line.partition('(')[2]
```

Comment:
This looks kinda verbose, I guess you could make better use of Python's regular expressions...